submissionForm.html

```diff
@@ -1,29 +1,32 @@
-<form action="/action_page.php">
-  <div class="container">
-    <label><b>Name</b></label>
-    <input type="text" placeholder="Enter Name" name="name" required>
-    <label><b>Last Name</b></label>
-    <input type="text" placeholder="Enter last name" name="lastname" required>
-    <label><b>Phone Number</b></label>
-    <input type="text" placeholder="Enter phone number" name="phonenumber" required>
-    <label><b>Email address</b></label>
-    <input type="text" placeholder="Enter email address" name="email" required>
-    <label><b>Home address</b></label>
-    <input type="text" placeholder="Enter home addres" name="homeaddress" required>
+<html>
+    <head>
+        <script type="text/javascript" src="submissionForm.js"></script>
+    </head>
+    <body>
+    <div class="container">
+        <label><b>Name</b></label>
+        <input type="text" placeholder="Enter Name" name="name" required>
+        <label><b>Last Name</b></label>
```

Comment:
since this is a small form and for the sake of showing you some anglar logic every element should have an unique id which will help you to grab the value and, check if empty and display it's own message.